.github/workflows/notion-sync.yaml

```diff
@@ -0,0 +1,29 @@
+name: Notion Sync
+
+on:
+  workflow_dispatch:
+  issues:
+    types:
```

Comment:
Can we sync PRs as well?

Reply:
No, the gha action code is just focused on issues and specifically ignores issues that are PRs (see [this line](https://github.com/tryfabric/notion-github-action/blob/master/src/sync.ts#L95-L97)).